manifests/kustomize/base/model-registry-service.yaml

```
@@ -3,6 +3,10 @@ apiVersion: v1
 metadata:
   labels:
     app: metadata
+    component: model-registry
```

My suggestion is to seize this chance to align label app and component in the Service here as:

```suggestion
    app: model-registry
    component: model-registry-server
```

to be consistent with the Deployment:

https://github.com/kubeflow/model-registry/blob/709a2dbb6ec387255c2a1cc7518d1edad07dad29/manifests/kustomize/base/model-registry-deployment.yaml#L2-L6

@Al-Pragliola @isinyaaa @dhirajsb wdyt ?

---

The app name is used to group related resources in a single app. E.g. the MR itself, it's deployment, service, istio resources, etc. should all use the same app name. 
So, we set it to the MR name downstream. The downstream labels are:
```
  labels:                                                                                                                
    app: modelregistry-sample                                                                                            
    app.kubernetes.io/component: model-registry                                                                          
    app.kubernetes.io/created-by: model-registry-operator                                                                
    app.kubernetes.io/instance: modelregistry-sample                                                                     
    app.kubernetes.io/managed-by: model-registry-operator                                                                
    app.kubernetes.io/name: modelregistry-sample                                                                         
    app.kubernetes.io/part-of: model-registry                                                                            
    component: model-registry                                                                                            
  name: modelregistry-sample                                                                                             
```

---

So I should add all of these labels?

---

@Al-Pragliola can you take up the task of consolidating the useful ones from here into a replacements yaml for an MR template for overlays?

---

@Griffin-Sullivan we can start with `app`, and `component` labels, we can definitely skip the operator label values as they don't apply. 
We should use [app.kubernetes.io](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) labels to be consistent with k8s recommendations.

---

@dhirajsb let me know what you think I added the changes.

---

lgtm @Griffin-Sullivan